frontend/app/actions/index.js

```diff
@@ -0,0 +1,27 @@
+export const REQUEST_BUSINESS_DETAIL = 'REQUEST_BUSINESS_DETAIL'
+export const RECEIVE_BUSINESS_DETAIL = 'RECEIVE_BUSINESS_DETAIL'
+
+export function requestBusinessDetail(business) {
+  return {
+    type: REQUEST_BUSINESS_DETAIL,
+    business
+  }
+}
+
+export function receiveBusinessDetail(business, json) {
+  return {
+    type: RECEIVE_BUSINESS_DETAIL,
+    business,
+    businessDetail: json,
+  }
+}
+
+export function fetchBusiness(business) {
+  return function (dispatch) {
+    dispatch(requestBusinessDetail(business))
+    return fetch('/api/v1/businesses/' + business['camis'])
+    .then(resp => resp.json())
+    .then(json =>
+      dispatch(receiveBusinessDetail(business, json)))
+  }
```

Comment:
Please add some basic error handling in the promise chain. Bootstrap has some nice, responsive classes for error messages so it should be pretty straightforward to just show the user a message if the server responds with a status code of 4xx or 5xx. It needs to be React / Redux friendly of course so you should look up the "proper" way to do this in your framework before implementing anything. Maybe this should be another ticket.